internal/codeintel/policies/matcher_common_test.go

```diff
@@ -99,21 +100,35 @@ func testUploadExpirerMockGitserverClient(defaultBranchName string, now time.Tim
 		return refs, nil
 	}
 
-	commitsUniqueToBranch := func(ctx context.Context, repo api.RepoName, branchName string, isDefaultBranch bool, maxAge *time.Time) (map[string]time.Time, error) {
-		branches := map[string]time.Time{}
-		for _, commit := range branchMembers[branchName] {
-			if maxAge == nil || !createdAt[commit].Before(*maxAge) {
-				branches[commit] = createdAt[commit]
+	commits := func(ctx context.Context, repo api.RepoName, opts gitserver.CommitsOptions) ([]*gitdomain.Commit, error) {
+		commits := []*gitdomain.Commit{}
```

Comment:
Small style thing, but is there a reason you don't use the var syntax here?

```suggestion
		var commits []*gitdomain.Commit
```

Reply:
No reason, what would be the difference, other than `commits` passing a `nil` check? Never fully understood when to use which 😬

Reply:
It avoids an allocation if the for loop below doesn't end up appending anything